packages/editor/src/components/post-actions/actions.js

```diff
@@ -623,22 +625,21 @@ const renamePostAction = {
 	},
 };
 
-function useRenamePostAction( resource ) {
+function useRenamePostAction( postType ) {
 	return useCanUserEligibilityCheckPostType(
 		'update',
-		resource,
+		postType,
 		renamePostAction
 	);
 }
 
 const useDuplicatePostAction = ( postType ) => {
-	const { userCanCreatePost } = useSelect(
+	const userCanCreatePost = useSelect(
 		( select ) => {
-			const { getPostType, canUser } = select( coreStore );
-			const resource = getPostType( postType )?.rest_base || '';
-			return {
-				userCanCreatePost: canUser( 'create', resource ),
-			};
```

Comment:
Prev code triggered a request to the `wp/v2` namespace.

## Screenshot
![CleanShot 2024-07-11 at 15 00 21](https://github.com/WordPress/gutenberg/assets/240569/4af169b6-f86c-47dc-8a87-5cf80b3121a9)

Reply:
So this is a 6.6 bug?

Reply:
Yes, but it shouldn't affect users. It just makes an extra request.

First, it will call `canUser( 'create', '' )`, and t hen when the post type object is resolved, it will call it correctly.

Reply:
Ah 👍

Reply:
Duplicate action is also not included in the core, it is in the plugin only for now, so it should not have an impact on 6.6.